sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -112,7 +112,25 @@ object JdbcUtils extends Logging {
    */
   def insertStatement(conn: Connection, table: String, rddSchema: StructType, dialect: JdbcDialect)
       : PreparedStatement = {
```

Comment:
Actually, this is an approach similar to `normalizePartitionSpec` in `PartitioningUtils.scala`.

https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/PartitioningUtils.scala#L248-L251